docs/credits.md

```diff
@@ -1 +1 @@
---8<-- "CREDITS.md"
```

Comment:
Should the file CREDITS.md be deleted from the repo now?

Reply:
Yes!